### DISCUSSION TITLE
Creating a spool folder for recieved .txt

### DISCUSSION THREAD

**Discussion by mbaggnx** [porrey/Virtual-ZPL-Printer#26] [Ideas]
Hello! hows going?

I would like to ask if it is possible to keep in another folder or in the same, the .txt archieve that is recieved on the spool

<img width="362" alt="image" src="https://user-images.githubusercontent.com/119416732/218470330-bef0955f-94d3-4305-ab94-876dbca500d3.png">

Thanks, kind regards

**Comment by porrey** [comment]
Yes, I will add this on the next release.
